app/src/main/java/swati4star/createpdf/activity/MainActivity.java

```diff
@@ -77,7 +77,6 @@
     private boolean mDoubleBackToExitPressedOnce = false;
     private Fragment mCurrentFragment;
 
-
```

Comment:
revert back this change

Reply:
in line 80 ??
it was just clear space

Reply:
i did set app name in home fragment because 
when user will go to another fragment it will change title
after he can get into home fragment with tow way 
1 is back button = it will change title to app name automatically 
2 is via navigation drawer = it's not changing title so i have used setTitle